src/models/usuario.js

```diff
@@ -19,31 +20,39 @@ const ProfesionalSchema = Schema({
         type: String,
         required: true
     },
+    img: {
+        type: String
+    },
+    profesional: {
+        type: Boolean,
+        required: true
+    },
     sector: {
         type: String,
         required: true
     },
+    descripcion: {
+        type: String,
+        required: true
+    },
     cuentaBancariaIBAN: {
         type: String,
         required: true
     },
     titularCuenta: {
         type: String,
         required: true
-    },
-    img: {
-        type: String
-    },
-    eventosId: [{
-        type: Schema.Types.ObjectId,
-        ref: 'Evento'
-    }]
+    }
+});
 
+usuarioSchema.pre('save', function(next) {
+    this.password = bcryptjs.hashSync(this.password, 10)
+    next()
+})
 
-});
-ProfesionalSchema.method('toJSON', function() {
-    const { __v, _id, password, ...object } = this.toObject();
-    object.uid = _id;
-    return object;
+usuarioSchema.method('compararPassword', async function(password) {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Backend/pullRequest?prid=7127635)
```suggestion
usuarioSchema.method("compararPassword", async function(password) {
```